### ISSUE TITLE
[Bug] Character slowly slides when standing on floors that aren't level with the ground

### ISSUE THREAD

**Issue by NetGlitched** [tsunamayo/Starship-EVO#3178]
### Short Description of the issue:

I started a new game and walked into one of the default ships that spawn in the tutorial area, and noticed very quickly that because the ship isn't completely level with the ground, my character just slowly slides across the floor when this really shouldn't be happening.

### Steps to reproduce the issue:

Land a ship so that it's slightly slanted, that should reproduce it.

### Environment
- Starship EVO version: 20w45a
- Graphics Card: GTX 1080

### Additional information:

No additional information should be necessary.

**Comment by tsunamayo** [comment]
Yes it is a known issue, it is somehow very hard to make this work while also allowing moving platform or entity to push the player. It will take time to get a fix. Thanks

**Comment by CurioInventorium** [comment]
Having friction for player entities that are colliding with other entities should help minimise the problem.
It also happens when the player is in two different gravity fields, e.g. The ring and on board a spaceship.
